content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -77,12 +77,20 @@ EndpointSlices support three address types:
 
 * IPv4
 * IPv6
-* FQDN (Fully Qualified Domain Name)
+* FQDN (Fully Qualified Domain Name; *deprecated*)
 
 Each `EndpointSlice` object represents a specific IP address type. If you have
 a Service that is available via IPv4 and IPv6, there will be at least two
 `EndpointSlice` objects (one for IPv4, and one for IPv6).
 
+{{< note >}}
+
+Setting the `addressType` for an EndpointSlice to the value "FQDN" is deprecated
+due to lack of clearly defined semantics. For more details, you can read the 
+[discussion](https://github.com/kubernetes/kubernetes/issues/114210).
```

Comment:
I'm not sure pointing people to the issue is the right thing.

I'd say that you shouldn't use it for any new purposes, but you might find existing third-party components that use it, in which case you should use it in the way that the third-party component says to use it.

Reply:
Aside: might be nice to have an off-by-default feature gate that warns a client when making an unowned EndpointSlice of this type.

(potentially even better, have `kubeadm` deploy a ValidatingAdmissionPolicy to do that, but this is a whole new kubeadm feature and therefore a lot more effort)